### PR TITLE
Remove schedule / agenda related sql code

### PR DESCRIPTION
**Pull request by cbonoz**
Still need to examine FAQ

### CODE REVIEW

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/ScheduleDatabase.java [hunk 1]
```diff
@@ -25,29 +25,6 @@
     private static SQLiteDatabase sDB;
```
Comment (kenyee) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/ScheduleDatabase.java]: Can this file just be deleted?  Or will we ever need a sqldb locally?

Comment (cbonoz) [reply Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/ScheduleDatabase.java]: FAQ still being retrieved from SQLite, but renamed the file to reflect this and moved the schedule items.

**Comment by cbonoz** [comment]
Possible to get re-review (added some changed) and merge if ok? Don't have write access.

**Comment by kenyee** [comment]
Looks good except for merge conflicts.
You should have merge capability..if you can create a branch, you should be able to merge it in git.

**Pull request merged**